### PR TITLE
Milestone 4: Basic UI

### PR DESCRIPTION
**Pull request by iKrystyan**
## In this milestone, the following features were added:
- Create a `app.rb` file that will serve as console app entry-point. It has methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- created `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of the file. This method does the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.
  
## Screenshot

image

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+  def create_rental
+    puts 'Select a book by its ID: '
+    @books.each_with_index do |book, index|
+      puts "ID: #{index}, Title: #{book.title}, Author: #{book.author}"
+    end
+    selected_book = gets.chomp.to_i
+    puts 'Select a person by ID: '
+    @people.each_with_index do |person, index|
+      puts "ID: #{index}, Name: #{person.name}, Age: #{person.age}"
```
Comment (MahmoudBakr23) [inline app.rb:108]: **Optional**
- Well done on the UI so far. However, it'd make more sense if you changed the person word with the rental work on listing all rentals for a given rental ID 😄

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,131 @@
+require './teacher'
+require './student'
+require './book'
```
Comment (MahmoudBakr23) [inline app.rb:3]: **Optional**
- As all of the required files are on the same dir, it'd make more sense if you used `require_relative`

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @Christianib003 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

**Highlights**
- Great work modifying the main file to handle the UI
- Well done modifying the app file to host all of the classes instances
- Nice job following the design
- Descriptive PR
- Correct Linters

**Optional**
- Please fix the MIT link for a professional README file
_Kindly see the comments under the review_

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (iKrystyan) [reply app.rb:108]: Thanks! @MahmoudBakr23

**Pull request merged**